test/other/codesize/test_codesize_minimal_pthreads.jssize

```diff
@@ -1 +1 @@
-8696
+8921
```

Comment:
Why did JS size increase here? The new musl code enabled here might be using a bunch of new JS perhaps?

Reply:
I optimized the wasi clock JS code a little so now this regression is gone.

Reply:
Great!